lib/config.ts

```diff
@@ -6,7 +6,6 @@ export class Configuration extends ConfigBase implements IConfiguration { // Use
 	CI_LOGGER = false;
 	DEBUG = false;
 	TYPESCRIPT_COMPILER_OPTIONS = {};
-	USE_PROXY = false;
```

Comment:
I believe a short article describing what is using a proxy good for and how to set it up, would be helpful for the users.